pyiceberg/manifest.py

```diff
@@ -1279,6 +1279,7 @@ def __init__(
                 "parent-snapshot-id": str(parent_snapshot_id) if parent_snapshot_id is not None else "null",
                 "sequence-number": str(sequence_number),
                 "format-version": "2",
+                "content": "data",
```

Comment:
This header is missing, and should be set to `data` until we support MoR deletes.